### ISSUE TITLE
S3uploader_downloader won't work in china/any region other than default

### ISSUE THREAD

**Issue by joshoftenly** [jswope00/s3-uploader-downloader#3]
TL:DR this doesn't work on an s3 bucket in the cn-north-1 aws region because the tool doesn't use regions in the session. 

firstly I want to say your all's tool is awesome, and that I'm sorry for bugging you guys, S3 is a rather critical resource for us. 

The xblock is working and we've verified that the bucket we configured is working on AWS CLI  from the server we have open edx running on. 

In your project boto (not boto3) is being called and doesn't have a session manager for region. Because our region is cn-north-1 on aws this is not working as it always shows "aws access key provided doesn't exist in our records". 

We worked on some hotfixes (editing boto directly) as well as making a function to insert region in your project code inside our instance which obviously isn't best practice and neither were working, I was wondering if you all would be able to provide a solution, obviously this is just hopeful question, or if you had resources on how to fix that that would be great as well. 

If you need access keys to an s3 bucket in cn-north-1 I will be happy to send them to your email. 

Thanks much for your time either way.

**Comment by jswope00** [comment]
Hi Josh, thanks for finding this and bringing it up. Sounds useful to be more flexible with regions. I'll have the team take a look and see if we can add this feature in the next week or so. Ping me end of next week if you don't hear from me by then.
